src/Common/src/System/IO/PathInternal.cs

```diff
@@ -98,8 +82,7 @@ internal static StringBuilder TrimEnd(this StringBuilder builder, params char[]
         internal static int FindFileNameIndex(string path)
```

Comment:
Where is this method used?  Can it just be deleted?

Reply:
@JeremyKuhne  should I delete this too ?

Reply:
If it isn't used anywhere go ahead and delete the method. I _think_ it is only used in CoreCLR.